### ISSUE TITLE
unused line of code?

### ISSUE THREAD

**Issue by kant111** [ethereum/ethereumj#1263]
I was trying to see if Block Miner runs an independent separate thread with a time period or is there something that triggers a mining? Meanwhile, I found the code below where I think it is unused unless there is some spring framework magic that I am unaware of?

https://github.com/ethereum/ethereumj/blob/develop/ethereumj-core/src/main/java/org/ethereum/mine/BlockMiner.java#L53

**Comment by mkalinin** [comment]
Yep, as far as I can see this executor is not used in any part of the codebase.
